README.md

```diff
@@ -1,2 +1,117 @@
-# rasenganjs-examples
-A bank of examples projects built with rasenganjs and other tools
+
+
+
+<img src="https://github.com/rasengan-dev/rasenganjs/raw/main/assets/cards/1.png" alt="Rasengan.js Templates" />
+
+# Rasengan.js Templates
+
+[![License](https://img.shields.io/badge/license-MIT-green.svg)](LICENSE)
+[![Version](https://img.shields.io/badge/version-1.0.0-blue.svg)](package.json)
+[![NPM Version](https://img.shields.io/npm/v/rasengan.svg)](https://www.npmjs.com/package/rasengan)
+
+Welcome to the Rasengan.js Templates repository! This repository contains a variety of templates to help you get started with your Rasengan.js projects quickly and efficiently.
+
+## Table of Contents
+
+- [Introduction](#introduction)
+- [Templates](#templates)
+- [Usage](#usage)
+- [What's Included](#whats-included)
+- [Contributing](#contributing)
+- [Present Contributors](#present-contributors)
+- [Resources](#resources)
+- [Communication](#communication)
+- [Code of Conduct](#code-of-conduct)
+- [License](#license)
+
+## Introduction
+
+Rasengan.js is a powerful framework for building modern web applications with ease. This repository provides pre-prepared templates that you can use to kickstart your projects. Whether you are building a chatbot, an e-commerce site, a portfolio, or a dashboard, we have a template for you.
+
+## Templates
+
+We currently offer the following templates:
+
+### Chatbot
+- **Gemini**
+  - **Description**: Gemini chatbot template, a ready-to-go chatbot app.
+  - **Path**: `templates/chatbot/gemini`
+  - **Language**: TypeScript
+
+### Portfolio
+- **Developer**
+  - **Description**: Developer portfolio template to showcase your projects to potential employers.
+  - **Path**: `templates/portfolio/developer`
+  - **Language**: TypeScript
+
+Additional templates are planned for various other categories, including e-commerce, dashboards, and more.
+
+## Usage
+
+To use a template, follow these steps:
+
+1. Clone the repository:
+   ```bash
+   git clone https://github.com/rasengan-dev/rasenganjs-examples.git
+   cd rasenganjs-examples
+   ```
+
+2. Install the Rasengan.js CLI:
+   ```bash
+   npm install -g rasengan@latest
+   ```
+
+3. Create a new project using a template:
+   ```bash
+   rasengan create <template-key>
+   ```
+
+   For example, to create a project using the Gemini chatbot template:
+   ```bash
+   rasengan create chatbot/gemini
+   ```
```

Comment:
This section has to be updated, it's not the right way to use templates in Rasengan.Js

What developers will do is:

- Explore the templates from a website **ex: templates.rasengan.dev**
- Identify the template that the developer want by copying its **template-key**
- Use the Rasengan CLI (without inslalling it explicitly) and generate a new project based on the template he has chosen. **ex: npx create-rasengan@latest [project-name] --template <template-key>** 

The **[project-name]** is not required in this command, so if the developer execute it without mentioning the [project-name] the CLI will ask him the name of the project later during the execution process.

Typically, that's the process.